platform.md

```diff
@@ -872,6 +873,9 @@ To achieve reproducibility the lifecycle SHOULD set the following to a constant,
 - file modification times in generated layers
 - image creation time
 
+The platform MAY set `SOURCE_DATE_EPOCH`  in the lifecycle execution environment, where the value of `SOURCE_DATE_EPOCH` MUST be a [UNIX timestamp](https://reproducible-builds.org/specs/source-date-epoch/).
```

Comment:
I think the additions to the `exporter` interface are sufficient and we do not need to describe this again here. This feature doesn't affect reproducibility as described here (same inputs -> same outputs) as `SOURCE_DATE_EPOCH` is an input